qualtran/_version.py

```diff
@@ -12,4 +12,4 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-__version__ = "0.1.0"
+__version__ = "0.2.0.dev0"
```

Comment:
Why `0.2.0.dev0` and not `0.2.0.dev` ?

Reply:
xref https://peps.python.org/pep-0440/#implicit-development-release-number